.github/workflows/code.yml

```
@@ -127,7 +127,8 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
 
       - name: Test cli works in runtime image
-        run: for i in ${{ steps.meta.outputs.tags }}; do docker run ${i} --version; done
+        # check that the first tag can run with --version parameter
+        run: docker run $(echo ${{ steps.meta.outputs.tags }} | sed -e 's/\s.*$//') --version
```

No idea how much of javascript is supported, does this work?
```suggestion
        run: docker run ${{ steps.meta.outputs.tags[0] }} --version
```

---

No it doesn't, but this works and is slightly cleaner
```suggestion
        run: docker run $(echo ${{ steps.meta.outputs.tags }} | head -1) --version
```

---

Yep, I like that and am embarrassed to have 'gone all sed' on this simple problem. Must have been a late night :-)